docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:22
+
+WORKDIR /app
+
+COPY package.json ./
+COPY mesh.config.ts ./
+
+RUN npm i
+
+EXPOSE 4000
```

Comment:
Regarding ports in general, I recommend checking: 
https://spacelift.io/blog/docker-security 
https://cheatsheetseries.owasp.org/cheatsheets/Docker_Security_Cheat_Sheet.html

We should use users and groups on the OS level and make sure the image we provide is safe/secured by default.